### ISSUE TITLE
Switching through tabs may lead to audio playback | maybe [feature]?

### ISSUE THREAD

**Issue by DrDragonKiller** [ParticleCore/Iridium#394]
Having these 2 options enabled:
_Play videos by hovering the thumbnails?
Shift key toggles audio on video thumbnail playback?_
and switching through tabs by using _Ctrl + Tab_ and _Ctrl + Shift + Tab_ may cause a video to play sound if you hovering on an thumbnail and wait on such page long enough.

This could be resolved by testing if while pressing Shift, Ctrl is pressed simultaneously (maybe even add an option)

**Comment by ParticleCore** [comment]
It is possible that this is a conflict due to YouTube's new feature: https://www.androidpolice.com/2018/03/19/youtube-picture-picture-now-tested-web/

Will have to investigate this further.

**Comment by ParticleCore** [comment]
I'm sorry, I thought I was able to reproduce this, but I was mistaken by another tab already playing in the background. I am not able to reproduce this issue, the video stops automatically when switching tabs using Ctrl + Tab and won't switch if I hold Shift together with those keys.

Please follow the troubleshooting guide as requested before opening a new issue: https://github.com/ParticleCore/Iridium/wiki/Report-a-bug

**Comment by ParticleCore** [comment]
Closing this for now, if you are still having trouble please follow the instructions in my previous reply.
